src/main/java/de/unijena/cheminf/fragment/fingerprint/performanceTest/Main.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2023 Betuel Sevindik, Felix Baensch, Jonas Schaub, Christoph Steinbeck, and Achim Zielesny
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package de.unijena.cheminf.fragment.fingerprint.performanceTest;
+
+/**
+ * Main class for starting the PerformanceTest application.
+ *
+ * @author Betuel Sevindik
+ */
+public class Main {
+    /**
+     * Starts the application. The first and second command line arguments must be
+     * the name of a text file to read (must be located in the same directory as the application's .jar file)
+     * Important for the correctness of the text files to be read is that both text files must start with a header.
+     * In addition, a molecule must be represented in each line of the text files and both text files must contain
+     * a ";" as separator. The first text file must contain a SMILES string at the first position of
+     * each line (all other positions are ignored). The first two positions of each line of the second file
+     * are ignored. These positions contain the molecule name/ID and the corresponds SMILES string. And all other
+     * positions within a line must alternately contain a SMILES string and an
+     * integer value separated by the delimiter (representing the frequency of the SMILES string).
+     *
```

Comment:
The detailed description of the CMD parameters should go to the PerformanceTest class because there, they are actually parsed. And maybe you could try describing this in a more clear way? Maybe do an enumeration-type structure of the three arguments, specifically (and briefly) stating what the arguments are used for and how the files among them need to be structured, e.g. semicolon-separated text files with header lines, first column must be a SMILES string in the first file (-> fragment structure for intialising the fingerprint), third column and following in the second file must be alternating pairs of fragment SMILES and frequencies in the respective molecule.